docs/en/source/installation/python-sdk.rst

```diff
@@ -1,113 +1,82 @@
+.. code-block:: sh
 
-  >> runtime.__full_version__
-  'Furiosa SDK Runtime .release:0.1.1+907338a44e91f176495b3c24fce3d9b1e626a662 (libnux 0.3.0-dev 9418048e4 2021-03-29 02:59:26)'
+  # Install the FuriosaAI NPU Python SDK to use the Python interface, e.g. `import furiosa`
+  pip install --upgrade furiosa-sdk~=0.1.0
+  # Install additional tools, see below for details
+  pip install --upgrade furiosa-sdk[runtime,quantizer]~=0.1.0
+  # Install all additional tools
+  pip install --upgrade furiosa-sdk[full]~=0.1.0
 
+The following additional packages can be installed by using ``pip``. 
 
-Inference can be executed by loading the model and creating a ``session`` object.
-When first loading the model, the process of compiling and optimizing the model is performed internally. 
-Initially, loading a model takes up to tens of seconds to compile and optimize an DNN model. 
-Once you load a model, ``session`` will become ready to execute inferences immediately.
+* ``cli``: command line tool installation, refer to :doc:`/quickstart/cli` for usage
 
-.. code-block::
+  .. code-block::
 
-  >>> sess = session.create('./MNISTnet_uint8_quant_without_softmax.tflite')
-  num_slices: 16
-  num_slices: 16
-  [1/6] 🔍   Compiling from tflite to dfg
-  Done in 0.001876285s
-  [2/6] 🔍   Compiling from dfg to ldfg
-  ▪▪▪▪▪ [1/3] Splitting graph...Done in 1.2222888s
-  ▪▪▪▪▪ [2/3] Lowering...Done in 0.24661668s
-  ▪▪▪▪▪ [3/3] Precalculating operators...Done in 0.007146129s
-  Done in 1.4769053s
-  [3/6] 🔍   Compiling from ldfg to cdfg
-  Done in 0.000111313s
-  [4/6] 🔍   Compiling from cdfg to gir
-  Done in 0.004966648s
-  [5/6] 🔍   Compiling from gir to lir
-  Done in 0.000273256s
-  [6/6] 🔍   Compiling from lir to enf
-  Done in 0.003491739s
-  ✨  Finished in 1.4908016s
+    pip install --upgrade furiosa-sdk[cli]
 
+* ``runtime``:  Install a number of libraries to accelerate models using the NPU using the FuriosaAI NPU Runtime, **Required for model acceleration using the NPU**
 
-After loading a model, you can print its information and see a list of input/output tensors and general information about each tensor.
+  .. code-block::
 
-.. code-block::
+    pip install --upgrade furiosa-sdk[runtime]~=0.1.0
 
-  >>> sess.print_summary()
-  Inputs:
-  {0: TensorDesc: shape=(1, 28, 28, 1), dtype=uint8, format=NHWC, size=784, len=784}
-  Outputs:
-  {0: TensorDesc: shape=(1, 1, 1, 10), dtype=uint8, format=NHWC, size=10, len=10}
+* ``quantizer``: Install the model quantization tool (see :doc:`/advanced/quantization`)
 
+  .. code-block::
 
-You can run the inference by calling ``session.run()``. ``run()`` 
-can take a list of `numpy.ndarray <https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html>`_
-or a single `numpy.ndarray` as arguments.
+    pip install --upgrade furiosa-sdk[quantizer]~=0.1.0
 
+* ``validator``: Installs model analysis tools, quantizes for accelerating corresponding models on the NPU, and includes compilation success verification tools. 
```

Comment:
I think we should parallelize these four bullet items.

Option 1)
\* \`\`cli\`\`: **Installs** the command line tool. Refer to :doc:\`/quickstart/cli\` for usage
\* \`\`runtime\`\`: **Installs** various libraries to accelerating models using the NPU using the FuriosaAI NPU Runtime. \*\*Required for model acceleration using the NPU\*\*
\* \`\`quantizer\`\`: **Installs** the model quantization tool (see :doc:\`/advanced/quantization\`)
\* \`\`validator\`\`: **Installs** model analysis tools, **quantizes** for accelerating corresponding models on the NPU, and **includes** compilation success verification tools. 

Option 2)
\* \`\`cli\`\`: **Installing** the command line tool. Refer to :doc:\`/quickstart/cli\` for usage
\* \`\`runtime\`\`: **Installing** various libraries to accelerating models using the NPU using the FuriosaAI NPU Runtime. \*\*Required for model acceleration using the NPU\*\*
\* \`\`quantizer\`\`: **Installing** the model quantization tool (see :doc:\`/advanced/quantization\`)
\* \`\`validator\`\`: **Installing** model analysis tools, **quantizing** for accelerating corresponding models on the NPU, and **including** compilation success verification tools.